apps/src/lib/levelbuilder/lesson-editor/LevelToken.jsx

```diff
@@ -161,6 +167,12 @@ export class LevelTokenContents extends Component {
             )}
             {activeLevel.bonus && <span style={styles.tag}>bonus</span>}
             {activeLevel.challenge && <span style={styles.tag}>challenge</span>}
+            {hasVariants && (
+              <span style={styles.inactiveVariants}>
+                inactive variant:&nbsp;
+                {inactiveLevels.map(level => `"${level.key}" `).join('\n')}
```

Comment:
Seems pretty unlikely but if there are multiple variants I think the level token might get too crowded? Is it worth just saying there is an inactive variant on the LevelToken using the purple things we use for challenge/assessment/bonus and then listing the inactive variants in the LevelTokenDetails?

Reply:
good point. the utility method does not let us add a second variant: https://github.com/code-dot-org/code-dot-org/blob/ff2d4fb572afedc8331a635f17875a0b663bac50/dashboard/app/models/script_level.rb#L711

however if we found that we needed to add one in an emergency, we'd certainly find a way to quickly work around that limitation.

Practically speaking I don't think this scenario is worth spending time on since we've never run into it in the past, and the workaround would be to make the window bigger to see all the variants.

I remember the purple things you're talking about, but I don't see them at the moment:
![Screen Shot 2021-06-03 at 9 52 50 AM](https://user-images.githubusercontent.com/8001765/120682766-af74e200-c451-11eb-8423-0a109ed78253.png)

Reply:
Ok I think that makes sense. I think going forward not worrying about the case of two-level variants is good. 

Figured out why the purple things are not showing: https://github.com/code-dot-org/code-dot-org/pull/40949. Don't know if you are concerned with something marked as assessment and having a level variant